AppCenter/AppCenterTests/MSDBStorageTests.m

```diff
@@ -694,6 +702,30 @@ - (void)testDatabaseThatIsAutoVacuumedNotManuallyVacuumedWhenInitialized {
   self.sut = [[MSDBStorage alloc] initWithSchema:self.schema version:0 filename:kMSTestDBFileName];
 }
 
+- (void)testDatabaseThatFileWasCorrupted {
+
+  // If
+  const char *validFileHeader = "SQLite format 3";
+  NSURL *fileURL = [MSUtility fullURLForPathComponent:kMSTestDBFileName];
+  NSData *data = [NSData dataWithContentsOfURL:fileURL];
+  
+  // Check that database file is valid.
+  XCTAssertEqual(strcmp((const char *)data.bytes, validFileHeader), 0);
```

Comment:
Any reason why we have to use `strcmp`?  Why wouldn't this work:

```suggestion
  XCTAssertEqual((const char *)data.bytes, validFileHeader);
```

Reply:
`XCTAssertEqual` doesn't compare C-strings

See https://developer.apple.com/documentation/xctest/xctassertequal

> An expression of **C scalar type**.